src/client/dfuse/dfuse_fuseops.c

```diff
@@ -128,7 +128,7 @@ df_ll_create(fuse_req_t req, fuse_ino_t parent, const char *name,
 
 	keep_ref = parent_inode->ie_dfs->dfs_ops->create(req, parent_inode,
 							  name, mode, fi);
-	if (!keep_ref) {
+	if (1 || !keep_ref) {
```

Comment:
Will keep_ref usage be part of a later behaviour ??

Reply:
you are not looking at the latest patch. i accidentally pushed this before reverting it as i was doing some investigation in to a memory leak. but i reverted that..